store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -5851,6 +5863,48 @@ public AddInviteData addInvite(OperationContext octxt, Invite inv, int folderId,
         }
     }
 
+    public String getMsgHtmlDesc(ParsedMessage pm) throws ServiceException {
+        String htmlDesc = null;
+        if (pm == null)
+            return htmlDesc;
```

Comment:
nit: try enclosing in curly braces.

Reply:
its a single line statement in if block, no need for curly braces

Reply:
Technically it looks good. That's why mentioned as nit.

Reply:
done